spec/migrations/20181016140921_migrate_orch_stacks_to_have_ownership_concept_spec.rb

```diff
@@ -0,0 +1,63 @@
+require_migration
+
+describe MigrateOrchStacksToHaveOwnershipConcept do
+  let(:orchestration_stack) { migration_stub(:OrchestrationStack) }
+  let(:ems) { migration_stub(:ExtManagementSystem) }
+  let(:service) { migration_stub(:Service) }
+  let(:tenant) { migration_stub(:Tenant).create }
+  let(:group) { migration_stub(:MiqGroup).create(:tenant => tenant) }
+  let!(:user) { migration_stub(:User).create(:userid => "admin", :miq_groups => [group]) }
```

Comment:
```suggestion
  let!(:user) { migration_stub(:User).create(:userid => "admin", :miq_groups => [group], :current_group => group) }
```

Alternative would be to add this to a `before_save` filter

```ruby
class User
  before_save :ensure_current_group

  def ensure_current_group
    current_group ||= miq_groups.first
  end
```